apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -442,13 +442,14 @@ const Arrow: React.FC = () => {
   );
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31277&commitId=2741b44871e8ff9e6841654f642a2c2e8aded70e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size overflow padding - RTL.chromium.png|0| Added|
|Positioning.auto size overflow padding shorthand.chromium.png|0| Added|
|Positioning.auto size overflow padding.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
expected